# This patch must also be applied to the source tree on the cross-compiling
# host when building crystal.o
Index: src/compiler/crystal/compiler.cr
--- src/compiler/crystal/compiler.cr.orig
+++ src/compiler/crystal/compiler.cr
@@ -507,6 +507,10 @@ module Crystal
           link_flags += " -L/usr/local/lib"
         end
 
+        if program.has_flag?("openbsd")
+          link_flags += " -L/usr/local/llvm18/lib"
+        end
+
         {DEFAULT_LINKER, %(#{DEFAULT_LINKER} "${@}" -o #{Process.quote_posix(output_filename)} #{link_flags} #{program.lib_flags(@cross_compile)}), object_names}
       end
     end
